src/gantry_calibration/calibration.py

```diff
@@ -0,0 +1,71 @@
+"""The core math and functionality behind the calibration algorithm."""
+
+import sys
+from collections import namedtuple
+import numpy as np
+
+
+Point = namedtuple("Point", ["x", "y"])
+PointPair = namedtuple("PointPair", ["expected", "actual"])
+
+
+class Calibration:
+    """Contains the calibration matrix and coefficients."""
+
+    # pylint: disable=too-few-public-methods
+
+    def __init__(self, x_order: int, y_order: int, points: list[list[PointPair]]) -> None:
+        """Instantiate the object with expected and actual points."""
+        self.x_order = x_order
+        self.y_order = y_order
```

Comment:
This code assumes all variables never get changed after the class is initialized. This is ok, but it would be more robust if the user could change the order and/or points after initialization and have the code re-check the order condition and re-calculate the mapping coefficients automatically.

Reply:
Refactored with your suggestions of using properties.